.github/workflows/deploy.yml

```diff
@@ -36,15 +36,14 @@ jobs:
             EOF
         - name: Setup
           run: |
-            # sudo apt update
-            # sudo apt install libgif-dev
+            sudo apt update
+            sudo apt install pkg-config libcairo2-dev libpango1.0-dev libssl-dev libjpeg62-dev libgif-dev
```

Comment:
Are we certain this is the complete set of dependencies? Just don't want to miss something!

Reply:
This is the list from [our README.md file](https://github.com/scratchfoundation/paper.js?tab=readme-ov-file#installing-native-dependencies-on-debianubuntu-linux), so if it's not sufficient then we should update both :)

(I'm pretty confident that it's sufficient)